index.js

```diff
@@ -1,19 +1,69 @@
 /* globals require, __dirname, process */
 
 const path = require('path');
-const fs = require('fs');
+const { promises: fs } = require('fs');
 const builder = require("adguard-hostlists-builder");
+const SVGParser = require('svg-parser');
+
+// build services.json from services folder
+(async() => {
+    try {
+        // variable to store json's objects
+        let servicesData = []
+        // array with all service names
+        const dirNames = await fs.readdir('./services');
+        // collect data from service files
+        const collectData = dirNames.map(async(dirName) => {
+            // get file path for reading data
+            const filePath = path.resolve(`./services/${dirName}`)
+            // get data from service file
+            const fileData = await fs.readFile(filePath)
+            // parse JSON string
+            const fileDataObject = JSON.parse(fileData);
```

Comment:
Please do the basic validation here

Reply:
what do you think about separate script for validation which can be run as github workflow and triggered by every push?
in this way data validity will be checked for every PR automatically, not on build stage after the merge